utils/wasm-gen/Cargo.toml

```
@@ -14,4 +14,5 @@ wasmprinter = "0.2"
 [dev-dependencies]
 rand = { version = "0.8.0", features = ["small_rng"] }
 wasmparser = { version = "0.93.0" }
+wat = "1.0.53"
```

The last version of `wat` is v1.0.56
```suggestion
wat = "1.0.56"
```